synapse/federation/federation_server.py

```diff
@@ -915,6 +915,10 @@ async def on_edu(self, edu_type: str, origin: str, content: dict):
         if not self.config.use_presence and edu_type == "m.presence":
             return
 
+        # Temporary patch to drop cross-user key share requests
+        if edu_type == "m.key_share_request":
```

Comment:
I though they were `m.room_key_request`? That is what i've been seeing a lot of in the DB

Reply:
Err... yes, not sure where the other name came from.